rust/src/array.rs

```diff
@@ -981,6 +981,10 @@ mod tests {
             binary_array.get_value(1)
         );
         assert_eq!("parquet", binary_array.get_string(1));
+        assert_eq!(binary_array.value_offset(0), 5);
```

Comment:
nit: can you swap the order for the arguments? the expected value should be on the LHS.

Reply:
Yes, good catch. Done!